clients/src/main/java/org/apache/kafka/clients/consumer/internals/CommitRequestManager.java

```diff
@@ -207,6 +226,232 @@ public NetworkClientDelegate.UnsentRequest toUnsentRequest() {
         }
     }
 
+    private class UnsentOffsetFetchRequest extends RequestState {
```

Comment:
How about renaming to `OffsetFetchRequestState`, and ditto for `OffsetCommitRequestState`.

Reply:
yap that sounds good!